assets/schema/ensemble_schema.json

```diff
@@ -1080,6 +1080,52 @@
         }
       }
     },
+    "Staggered-Grid-payload": {
```

Comment:
this is not enough - you need to use it also. Look at any widget e.g. Button

Reply:
@vusters Yup, I updated it. Can you check the new changes?